shell/common/shell.h

```diff
@@ -528,10 +529,14 @@ class Shell final : public PlatformView::Delegate,
   void OnFrameRasterized(const FrameTiming&) override;
 
   // |Rasterizer::Delegate|
-  fml::Milliseconds GetFrameBudget() override;
+  fml::TimePoint GetLatestFrameTargetTime() const override;
 
   // |Rasterizer::Delegate|
```

Comment:
Is this stale?

Reply:
I'm not sure which line you're referring to - I moved this member up from the Rasterizer delegate to the CompositorContext delegate (Which the rasterizer delegate is now a subclass of).

Reply:
Ah, okay. I was just confused by the two `// |...|` comments, and thought one of them might be extra.